kong/plugins/acme/handler.lua

```diff
@@ -70,28 +70,33 @@ local domains_matcher
 -- expose it for use in api.lua
 ACMEHandler.build_domain_matcher = build_domain_matcher
 
-function ACMEHandler:init_worker()
-  local worker_id = ngx.worker.id()
-  kong.log.info("acme renew timer started on worker ", worker_id)
-  ngx.timer.every(86400, client.renew_certificate)
 
-  -- handle cache updating of domains_matcher
-  kong.worker_events.register(function(data)
-    if data.entity.name ~= "acme" then
-      return
-    end
+local CONFIG
 
-    local operation = data.operation
 
-    if operation == "create" or operation == "update" then
-      local conf = data.entity.config
-      domains_matcher = build_domain_matcher(conf.domains)
-    end
+local function renew(premature)
+  if premature or not CONFIG then
+    return
+  end
+  client.renew_certificate(CONFIG)
+end
 
 
-  end, "crud", "plugins")
+function ACMEHandler:init_worker()
+  local worker_id = ngx.worker.id()
+  kong.log.info("acme renew timer started on worker ", worker_id)
+  ngx.timer.every(86400, renew)
+end
+
+
+function ACMEHandler:configure(configs)
+  CONFIG = configs and configs[1] or nil
```

Comment:
Is this fine as we'll have only one ACME plugin that is being `configure`d? due to how our plugin iterator works?

Reply:
Answer: yes